docs/non_containerized_NSFS.md

```diff
@@ -228,7 +244,7 @@ Health status of the NSFS can be fetched using the command line.
  ```
 
 ## Bucket and Account Manage CLI
-Users can create, update, delete, and list buckets and accounts using CLI. If the config directory is missing CLI will create one and also create accounts and buckets sub-directories in it and default config directory is `/etc/noobaa.conf.d`. 
+Users can create, update, delete, and list buckets and accounts using CLI. If the config directory is missing CLI will create one and also create accounts and buckets sub-directories in it and default config directory is `${config.NSFS_NC_NOOBAA_CONF_DIR}`. 
```

Comment:
is its `${config.NSFS_NC_DEFAULT_CONF_DIR}` instead of `${config.NSFS_NC_NOOBAA_CONF_DIR}` ?

Reply:
thanks